packages/nitro-protocol/contracts/examples/RockPaperScissors.sol

```diff
@@ -16,69 +17,79 @@ import '../Outcome.sol';
   *
 */
 contract RockPaperScissors is ForceMoveApp {
+    using SafeMath for uint256;
 
-    enum PositionType { Start, RoundProposed, RoundAccepted, Reveal }
-    enum Weapon { Rock, Paper, Scissors }
+    enum PositionType {Start, RoundProposed, RoundAccepted, Reveal}
+    enum Weapon {Rock, Paper, Scissors}
 
-    struct RPSData{
+    struct RPSData {
         PositionType positionType;
         uint256 stake;
         bytes32 preCommit;
-        Weapon bWeapon;
-        Weapon aWeapon;
+        Weapon playerFirstWeapon;
```

Comment:
Maybe `playerOneWeapon` and `playerTwoWeapon` would be better. In fact, we should probably use `playerAWeapon` and `playerBWeapon`, since the app uses that nomenclature.

Reply:
I personally think `playerA` and `playerB` can be a bit confusing.
The reason for `playerFirst` is that in this turn based game `playerFirst` is the player who initiates the game.
If the DApp also uses this terminology do you think we should just stick with `A` and `B` or maybe refactor there as well?

Reply:
How about we stick with `A` and `B` for now, and revisit once the DApp is working again. I think at this stage it will cause us more pain to introduce a new nomenclature (even if it is superior).